docs/azure_arc_jumpstart/azure_arc_k8s/day2/multi_distributions/calico/_index.md

```diff
@@ -80,39 +82,45 @@ The policy will be set to “Audit” mode to check the configuration of existin
 
     ![AzResourceProvider Bash](./04.png)
 
-- Azure Policy installed in Azure Arc-Connected Cluster
+- Check whether Azure Policy is installed in an Azure Arc connected cluster
```

Comment:
Please update to Azure Arc-enabled Kubernetes cluster

Reply:
done